README.md

```diff
@@ -82,7 +82,10 @@ Open or comment on one of [our issues](https://github.com/ME-ICA/tedana/issues)!
 
 We ask that all contributions to ``tedana`` respect our [code of conduct](https://github.com/ME-ICA/tedana/blob/master/CODE_OF_CONDUCT.md).
 
-## Contributors
```

Comment:
I think we still want this heading, just below the quick start recommendation :smile_cat: 

Also, WDYT about moving up this text to above the code of conduct reminder ? That way it's clear that the CoC governs gitter, too. At least, that's how I'd read it !

Reply:
Wow, I totally missed this comment until @tsalo pointed it out, my bad. The subtraction was another merge casualty; I really need to be more careful next time. 

What would you think about additionally adding a line to clarify that the CoC applies broadly across the project, just to be super-duper clear? I (optimistically) hope it never becomes an issue, but at least then it's clear.